src/prefect/run_configs/vertex.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union, Iterable
+
+import yaml
+
+from prefect.run_configs.base import RunConfig
+from prefect.utilities.filesystems import parse_path
+
+
+class VertexRun(RunConfig):
+    """Configure a flow-run to run as an ECS Task.
+
+    Vertex Tasks are composed of and runtime parameters.
+
+    Runtime parameters can be specified via `run_task_kwargs`. These will be
+    merged with any runtime parameters configured on the agent when starting
+    the task.
+
+    Args:
+        - service_account (str):
+            Specifies the service account for workload run-as account.
+            Users submitting jobs must have act-as permission on this
+            run-as account. If unspecified, the `AI Platform Custom Code
+            Service
+            Agent <https://cloud.google.com/vertex-ai/docs/general/access-control#service-agents>`__
+            for the CustomJob's project is used.
```

Comment:
Do we want to consider implementing pulling a default service account from the environment? As is, we'll have to boilerplate in our general use service account with each call to the RunConfig. If we pull from the environment, we can set up that variable when we deploy the agents.

Reply:
Essentially, without additional terraform-fu, our current plan envisions only providing the central project the ability to access a single service account in the team project for spinning up vertex jobs anyway. As I understand it, this won't usually vary from job to job on a single agent.

Reply:
this is a great point. How about we put this service account as a property on the agent? And then this will override but it will pull the default from the agent

Reply:
this is done